### PR TITLE
feat(common): CSVRenderer: parse header labels

### PR DESCRIPTION
**Pull request by obdulia-losantos**
Continues https://jira.ehealthafrica.org/browse/AET-125

Include in the url the param `parse_columns` with the rules to generate csv headers. See tests for more examples.

In this use case: `parse_columns=remove-prefix:payload.`

For CHAMPS: `rule_sep=$&parse_columns=remove-prefix$payload.,replace$.$:`

### CODE REVIEW

File: aether-common-module/aether/common/drf/renderers.py [hunk 1]
```diff
@@ -40,7 +37,89 @@ def render(self, data, media_type=None, renderer_context={}, *args, **kwargs):
...
+            return value[len(prefix):]
+        return value
+
+    if rule.startswith('remove-suffix:') and rule != 'remove-suffix:':
+        suffix = rule.split(':')[1]
+        if value.endswith(suffix):
+            return value[:-len(suffix)]
+        return value
+
+    if rule.startswith('replace:'):
```
Comment (admbtlr) [inline aether-common-module/aether/common/drf/renderers.py]: Wow, this is amazing, you actually read my mind... I'd forgotten that the final thing we need to do for CHAMPS to change the '.' separator on the column headings into a ':' 

💯!

File: aether-common-module/aether/common/drf/renderers.py [hunk 2]
```diff
@@ -40,7 +37,89 @@ def render(self, data, media_type=None, renderer_context={}, *args, **kwargs):
...
+        old_sep = rule.split(':')[1]
+        new_sep = rule.split(':')[2]
+        return value.replace(old_sep, new_sep)
+
+    if rule.startswith('split:') and rule != 'split:':
```
Comment (admbtlr) [inline aether-common-module/aether/common/drf/renderers.py]: Is this any different to `replace:<sep>: `?

**Review by admbtlr — CHANGES_REQUESTED** [review]

File: aether-common-module/aether/common/drf/renderers.py (hunk 2 above)
Comment (obdulia-losantos) [reply aether-common-module/aether/common/drf/renderers.py]: Nop, just a shortcut of `replace:<sep>: :` :thinking:

File: aether-common-module/aether/common/drf/renderers.py (hunk 1 above)
Comment (obdulia-losantos) [reply aether-common-module/aether/common/drf/renderers.py]: In this case I'm not sure if I should use another rule separator like... `|` maybe

File: aether-common-module/aether/common/drf/renderers.py (hunk 2 above)
Comment (admbtlr) [reply aether-common-module/aether/common/drf/renderers.py]: OK cool, just checking

File: aether-common-module/aether/common/drf/renderers.py (hunk 1 above)
Comment (admbtlr) [reply aether-common-module/aether/common/drf/renderers.py]: `\:`?

Comment (obdulia-losantos) [reply aether-common-module/aether/common/drf/renderers.py]: I meant `replace|.|:` I'm quite sure that there are no `|` in the object keys :crossed_fingers:

Comment (obdulia-losantos) [reply aether-common-module/aether/common/drf/renderers.py]:
```
self.assertEqual(apply_label_rules(['replace:.:\:'], 'a.bcd'), 'a:bcd')

AssertionError: 'a\bcd' != 'a:bcd'
- a\bcd
+ a:bcd
```

**Review by admbtlr — APPROVED** [review]

**Pull request merged**